include/fiction/algorithms/simulation/sidb/detect_bdl_wires.hpp

```diff
@@ -597,11 +597,13 @@ class detect_bdl_wires_impl
      * lengths are found.
      *
      * @param type The type of the cell to filter by.
+     * @param forbidden_type The type of cell that is forbidden in the wire and is automatically filtered out.
      * @return A vector of `bdl_wire` objects containing cells of the specified type. If no such wires are found,
      *         an empty vector is returned.
      */
     [[nodiscard]] std::vector<bdl_wire<Lyt>>
-    filter_wires_by_type(const typename technology<Lyt>::cell_type& type) const noexcept
+    filter_wires_by_type(const typename technology<Lyt>::cell_type& type,
+                         const typename technology<Lyt>::cell_type  forbidden_type) const noexcept
```

Comment:
What if I want to prohibit multiple types?

Reply:
To be honest, I don't see a use for that at the moment.